src/pages/commits/ViewToggle.tsx

```diff
@@ -0,0 +1,90 @@
+import { useQuery } from "@apollo/client";
+import styled from "@emotion/styled";
+import {
+  SegmentedControl,
+  SegmentedControlOption,
+} from "@leafygreen-ui/segmented-control";
+import Tooltip from "@leafygreen-ui/tooltip";
+import { Body } from "@leafygreen-ui/typography";
+import Icon from "components/Icon";
+import { size } from "constants/tokens";
+import {
+  ProjectHealthView,
+  ProjectHealthViewQuery,
+  ProjectHealthViewQueryVariables,
+} from "gql/generated/types";
+import { PROJECT_HEALTH_VIEW } from "gql/queries";
+import { useQueryParam } from "hooks/useQueryParam";
+import { ProjectFilterOptions } from "types/commits";
+
+type Props = {
+  identifier: string;
+};
+
+export const ViewToggle: React.VFC<Props> = ({ identifier }) => {
```

Comment:
I think it could be worth adding an analytics events on this component to see how often people are interacting with it!